config/credentials/test.yml.enc

```diff
@@ -0,0 +1 @@
+a5tpdomnpFpGActY5TChOcKJpdxRrHDiDHqlx0+kFTWA0pmOv52st/iXw1jIKHAwxdpgGlDsGNbEIUJrCK6MZWA9rHw5KUAGmCgURZwdWGRY8+AL9yyuj060Njyl7FKwQ8XpbIGJqDZEgTRF0VDdH0Z53rv1RLO5ZPDTOHxS--qDqpJdDl9iVgNgpt--4PvYfOcmIbusDQxFDFcgrg==
```

Comment:
leave empty line at end of files

Reply:
This files are auto generated and they are not editable since are the product of an ecription process